.gitignore

```diff
@@ -15,7 +15,14 @@ result-*
 .node
 .node-cfg
 .projectile
-/dist/
+/dist/*
+!/dist/esbuild/
+/dist/esbuild/*
+!/dist/esbuild/index.html
```

Comment:
Please do not keep any files in dist. The name suggests it should be for build artifacts. It's OK to copy some data from e.g. `static` to `dist`.